src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -571,7 +571,7 @@ protected void VisitLvalue(BoundExpression node)
                 case BoundKind.PropertyAccess:
                     var access = (BoundPropertyAccess)node;
 
-                    if (Binder.AccessingAutoPropertyFromConstructor(access, _symbol))
+                    if (Binder.IsPropertyAssignableFromConstructor(access, _symbol)) // PROTOTYPE(semi-auto-props): Revise this method call is the behavior we want and add unit tests..
```

Comment:
>IsPropertyAssignableFromConstructor [](http://example.com/codeflow?start=31&length=35)

The name doesn't really convey anything about equivalence to access the backing field. I think it would be good to reflect that in the name. This place is specifically about the write access, I think.